frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/pipelines/pipeline-as-code.ts

```diff
@@ -13,23 +13,26 @@ import {
 } from '@console/dev-console/integration-tests/support/pages';
 import { pipelineActions, pipelineTabs, repositoryDetailsTabs } from '../../constants';
 import { createRepositoryPO } from '../../page-objects';
-import { pipelinesPage, repositoryDetailsPage, repositoriesPage } from '../../pages';
+import {
+  pipelinesPage,
+  repositoryDetailsPage,
+  repositoriesPage,
+  pipelineRunDetailsPage,
+} from '../../pages';
 import { actionsDropdownMenu, tableFunctions } from '../../pages/functions/common';
 
 Given('user has installed pipelines as code', () => {
   // Steps to install pipeline as code feature : https://gist.github.com/chmouel/272df2cfbeef5606ca36d29a9a2d2f96
-  cy.exec(
-    `oc apply -f https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/release-0.1/release-0.1.yaml`,
-    {
-      failOnNonZeroExit: false,
-    },
-  );
+  cy.exec(`oc apply -f ${'testData/repository-crd-testdata/pac-installation-release0.1.yaml'}`, {
+    failOnNonZeroExit: false,
+  });
   cy.exec(`oc expose service el-pipelines-as-code-interceptor -n pipelines-as-code`, {
     failOnNonZeroExit: false,
   });
 });
 
 Given('user is at repositories page', () => {
+  cy.reload();
```

Comment:
is this necessary?

Reply:
When we use oc apply, sometimes it will not show repository tab, for that purpose we need to reload the page to get that visible.

Reply:
@karthikjeeyar Is it possible that we watch something here so that the tabs appear?

Reply:
We do have watchers in place and Repository tabbed is feature flagged. So once it is installed it will automatically appear.